examples/sleepy-demo/app.c

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
```

Comment:
You're correct,  I just copied the sources from the GSDK and ran the `make-pretty` script on them. Functionally, the sources are identical to the GSDK.

We can definitely switch to just referring to the sources from the GSDK (See https://github.com/SiliconLabs/ot-efr32/pull/569). The reason I copied sources instead of referring to the ones in the GSDK is that having copies will allow us to push any bugfixes instead of having to wait for a new GSDK patch release. 

If you think the maintenance pain outweighs the potential benefits, I'm not opposed to removing the copied sources and just referring to the GSDK sources

Reply:
Good point re bug fixes.  Let's keep the 2 copies.